src/model/atom.jl

```diff
@@ -8,14 +8,43 @@ export
     setcharge, setstereo, atomnumber, todict
 
 
-const ATOMTABLE = YAML.load(open(
-    joinpath(dirname(@__FILE__), "../../assets/const/atomicweights.yaml")
-))
+const ATOMTABLE = let
+    weightsfile = joinpath(dirname(@__FILE__), "../../assets/const/atomicweights.yaml")
+    include_dependency(weightsfile)
```

Comment:
Using `include_dependency` will cause the package to be recompiled if the data change. Seemed like a useful addition to me.

Reply:
I did not know that, it looks very useful. Thank you!